### DISCUSSION TITLE
mode: 'jit' - "@apply is not supported within nested at-rules like @screen"

### DISCUSSION THREAD

**Discussion by knynkwl** [tailwindlabs/tailwindcss#4990] [Help] [answered]
When JIT is enabled, I get the error below. but if I run normally without jit it compiles fine.

```sh
CssSyntaxError: tailwindcss: /**/**/**/file.css:12:5: 
@apply is not supported within nested at-rules like @screen. 
We suggest you write this as @apply md:bg-black instead.
```

```js
postcss.config.js
module.exports = {
  parser: 'postcss-scss',
  plugins: {
    'postcss-partial-import': {},
    'tailwindcss': {},
    'postcss-preset-env': {
      'stage': 3,
      'features': {
        'nesting-rules': true,
      },
    },
    'postcss-focus-visible': {},
    'autoprefixer': {},
  }
}
```

**Comment by Florian-crg** [comment]
Same here, if I add   `mode: 'jit',` to my tailwindcss config I get stuck on this error !  Any fix which is not to refacto everything 
 in a dirty way :/ ?

**Reply by adamwathan** [reply]
Setup nesting using our nesting plugin and it will work:

https://tailwindcss.com/docs/using-with-preprocessors#nesting

**Reply by knynkwl** [reply]
Oh shit! Nice, maybe a small note about this on "enabling JIT" :)

**Comment by knynkwl** [comment, marked as answer]
Enabling `'tailwindcss/nesting'` in replace of `postcss-nested` or `postcss-preset-env` fixed this!
